noggin/tests/unit/controller/test_user_otp.py

```diff
@@ -20,14 +20,14 @@
 def dummy_user_with_2_otp(client, logged_in_dummy_user, dummy_user_with_otp):
     ipa = logged_in_dummy_user
     result = ipa.otptoken_add(
-        ipatokenowner="dummy",
-        ipatokenotpalgorithm="sha512",
-        description="dummy's other token",
-    )
-    token = OTPToken(result)
+        a_ipatokenuniqueid=None,
+        o_ipatokenowner="dummy",
+        o_ipatokenotpalgorithm="sha512",
+        o_description="dummy's other token",
+    )['result']
     yield (dummy_user_with_otp, OTPToken(result))
     try:
-        ipa_admin.otptoken_del(token.uniqueid)
+        ipa_admin.otptoken_del(result['dn'][0])
```

Comment:
Same here, it looks like `otptoken_del` takes the uniqueid as argument, not the dn.